packages/remove-bins/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@pnpm/remove-bins",
+  "version": "1.0.0",
```

make it 0.0.0 because the publish tool will bump it
```suggestion
  "version": "0.0.0",
```